# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-clang-includepath.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-134.0.6998.88/build/config/compiler/BUILD.gn.vanilla	2025-03-12 18:03:52.836871421 +0100
+++ chromium-134.0.6998.88/build/config/compiler/BUILD.gn	2025-03-12 18:05:36.921948338 +0100
@@ -273,7 +273,7 @@
 
 config("compiler") {
   asmflags = []
-  cflags = []
+  cflags = [ "T2_CLANG_INCLUDE_PATH" ]
   cflags_c = []
   cflags_cc = []
   cflags_objc = []
